srcpkgs/folks/patches/disable-failing-tests.patch

```diff
@@ -0,0 +1,53 @@
+diff --git a/tests/bluez/meson.build b/tests/bluez/meson.build
```

Comment:
Can you explain why these tests fail?

Reply:
tests fail in ci due to the timeouts.
x86_64
```
Summary of Failures:

44/44 folks:DummyBackend / search-view             TIMEOUT        30.01s
```
i686
```
Summary of Failures:

33/44 folks:BluezBackend / device-properties       TIMEOUT        30.01s
34/44 folks:BluezBackend / vcard-parsing           TIMEOUT        30.01s
35/44 folks:Telepathy / init                       TIMEOUT        30.01s
```

Reply:
test timeouts in the ci were fixed by disabling parallelism.

Reply:
Tysm for investigating.